set.nix

```diff
@@ -45,6 +45,10 @@ rec {
   */
   map = builtins.mapAttrs;
 
+  /* mapToList :: (key -> value -> value) -> set -> [value]
```

Comment:
the type is too restrictive (only as documented), no? i believe it could be `(key -> value -> a) -> set -> [a]`